cli/tests/pcluster/models/test_imagebuilder.py

```diff
@@ -48,7 +48,11 @@
                             {"key": "date", "value": "2022.1.1"},
                         ],
                     },
-                    "build": {"parent_image": "ami-0185634c5a8a37250", "instance_type": "c5.xlarge"},
+                    "build": {
+                        "imds": {"require_imds_v2": True},
```

Comment:
Do we cover the IMDSv1 scenario somewhere else?
I think it would be better to cover both cases: IMDSv1 and IMDSv2, especially because with this solution we are going to cover the non default scenario.
I know that currently these unit tests do not support templatization of cluster config, but we should add it sooner or later. May you please keep track of this improvement in our backlog?

Reply:
This specific test is for testing the key id encrypted and AMI volume size validations, the reason we added this bit of configuration was to avoid the warning for IMDSv2 since testing that behaviour was outside the scope of the test.